src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -122,10 +124,13 @@ function processElement(cellElement) {
 			if (lifeLoss >= critLifeThreshold || npcName == 'Undaron') {
 				$(npcFastAttackElement).css('color', '#E7971F');
 			}
-		//	$(npcFastAttackElement).append(' (-' + lifeLoss + ' LP)');
 			$(npcNameElement).addClass('processedNPC knownNPC');
-            $(npcFastAttackElement).css('color', '#006400');
-            $(npcFastAttackElement).append(' - LS: ' + lifeLoss + ' - NL: ' + (playerExpectedLife - lifeLoss));
+			$(npcFastAttackElement).css('color', '#006400');
+			$(npcFastAttackElement).append(' ( -' + lifeLoss + ' LP, = ' + (playerExpectedLife - lifeLoss) + ' LP )');
+			const randomWait = Math.floor(Math.random() * (2000 - 500 + 1) + 500);
+			setTimeout(function() {
+				// $(npcFastAttackElement).trigger('click'); - use this at your own risk...
+			}, randomWait);
```

Comment:
debugging leftover? might want to remove 😆

Reply:
I already thought you would ask me, no, no debbuging leftover :)

Check this >

in the routine;

	`initIgnoreNpcData();`

in the processElement;

		} else if (initIgnoreNpcData(npcName)) {
			// nothing

and

```
			const randomWait = Math.floor(Math.random() * (2000 - 500 + 1) + 500);
			setTimeout(function() {
				// $(npcFastAttackElement).trigger('click'); - use this at your own risk...
			}, randomWait);
```

and:

```
function initIgnoreNpcData() {
	// Ignore
	npcData['Sprühregenwurm'] = [3, 25];
	npcData['Alter Mann'] = [1, 3];
	npcData['Undaron'] = [1, 10];
}
```

i think, you know what this is :)

uncommenting this: 				`// $(npcFastAttackElement).trigger('click'); - use this at your own risk...`

= auto attack, and with the ignore function there is a small fallback if the npc is tooo strong > "use this at your own risk..."

Reply:
i just dont really get why u want to have commented out code in the script. right now, with it commented out, it doesnt execute. so its useless.

is this like an optional feature u want to keep in? then it should maybe be controlled via a `bool` instead, that one can set somewhere at the beginning of the script. `activateAutoAttack = true` ... `if (activateAutoAttack) ...`

Reply:
danke auch für die tollen erweiterungen 👍